middleware/packet-forward-middleware/packetforward/types/keys.go

```diff
@@ -3,7 +3,7 @@ package types
 import fmt "fmt"
 
 const (
-	// ModuleName defines the 29-fee name
+	// ModuleName defines the module name
 	ModuleName = "packetfowardmiddleware"
```

Comment:
I know previously we avoided correcting the typo found in the `ModuleName` because iirc it would then involve in place store migrations. If I'm understand everything correctly this PR would mean migrations have to take place anyway, do we think its worth fixing this type now?

Reply:
yes this is a good time to do so

Reply:
We actually can never rename this. It breaks denoms since it depends on the Module name for the GetReceiver func. RIP

Would require a big upgrade migration to transfer denoms to new types etc. :(((

Reply:
ahh good catch, i forgot that we are now making use of it in this context. not the end of the world, thanks for looking into it!